<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartForm</class>
 <widget class="QWidget" name="StartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>734</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThreeMonkey</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>79</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Wawati TC</family>
          <pointsize>56</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 56pt &quot;Wawati TC&quot;;</string>
        </property>
        <property name="text">
         <string>三不猴</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Wawati SC&quot;;</string>
        </property>
        <property name="text">
         <string>我就是我，最冷漠的旁观者</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Wawati SC&quot;;</string>
        </property>
        <property name="text">
         <string>游戏中请你尽量不要多管闲事，非礼勿听，非礼勿看，非礼勿言</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="learn">
        <property name="minimumSize">
         <size>
          <width>232</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Wawati SC&quot;;</string>
        </property>
        <property name="text">
         <string>· 进入游戏说明阶段（手势学习）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Wawati SC</family>
          <pointsize>24</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Wawati SC&quot;;</string>
        </property>
        <property name="text">
         <string>· 直接进入游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
